aptos-move/block-executor/src/executor.rs

```diff
@@ -359,7 +328,7 @@ where
         }
     }
 
-    fn worker_commit_hook(
```

Comment:
is post-commit post processing of the outputs, than the commit, commit seems more natural to refer to synchronization to know something is truly finalized, which has happened already (by the coordinator).

Reply:
I'll rename all *hook functions - they are not really `hooks` anymore. perhaps: prepare_commits/process_commits?